snmpwalk/manifest.json

```
@@ -26,22 +26,5 @@
     "sales_email": "help@datadoghq.com",
     "name": "Community"
   },
-  "oauth": {},
-  "assets": {
-    "integration": {
-      "source_type_name": "Snmpwalk",
-      "configuration": {},
-      "events": {
-        "creates_events": false
-      },
-      "metrics": {
-        "metadata_path": "metadata.csv",
-        "check": [],
-        "prefix": "snmpwalk."
-      },
-      "service_checks": {
-        "metadata_path": "assets/service_checks.json"
-      }
-    }
-  }
+  "oauth": {}
```

```suggestion
  "oauth": {},
  "assets": {}
```
Looks like the assets key is mandatory? ([docs](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags))